src/main/java/mate/academy/hibernate/relations/Main.java

```diff
@@ -15,18 +21,24 @@ public static void main(String[] args) {
         // use this session factory when you will initialize service instances
         SessionFactory sessionFactory = HibernateUtil.getSessionFactory();
 
+        CountryDaoImpl countryDao = new CountryDaoImpl(sessionFactory);
```

Comment:
Direct instantiation of `CountryDaoImpl` with `new` should be avoided. Instead, use dependency injection to provide the instance, which allows for better decoupling and easier testing.